### DISCUSSION TITLE
After extension install ulauncher dont run anymore

### DISCUSSION THREAD

**Discussion by lokize** [Ulauncher/Ulauncher#1410] [General]
Hi guys, after installing a extension ulauncher dont run anymore, how to restore default configurations?

**Comment by friday** [comment]
If this happens to you in any application, figure out which command to run (`ulauncher` in this case) and run it in the terminal. Then include the terminal log for the error when you search or ask for help.

(there are other ways to get the log output too, but this is the easiest way).

**Comment by lokize** [comment]
![image](https://github.com/user-attachments/assets/f3b191e1-e67b-4d2a-949b-01f9e19f4573)

---------------EDIT-------------------

I managed by opening it through the terminal and disabling the problematic extension!

**Reply by friday** [reply]
That crashes the extension, but not Ulauncher. What does it say at the end of the log where ulauncher crashes?

**Reply by friday** [reply]
This is not an error from Ulaunchers side. We expect and handle it, and it's also not an error from the extension devlopers side. This happens because you didn't follow the extension instructions: https://github.com/brpaz/ulauncher-vscode-projects?tab=readme-ov-file#install

**Comment by lokize** [comment]
It's working fine again. Taking advantage of the post, my Ulauncher always takes a while to open, and there seems to be a graphical glitch when loading. Is this normal?
[screen_record_20240905_113209.webm](https://github.com/user-attachments/assets/1e85e058-fab2-4695-a97d-8d288b3a6976)

**Reply by friday** [reply]
I've never seen that before, but I don't use KDE.

**Reply by friday** [reply]
Either way it's sure to be an issue between the libraries Ulauncher uses, your DE, your graphics drivers, and/or your hardware. Probably not something Ulauncher can fix.

**Reply by lokize** [reply]
Got it, I managed to fix the VSCode extension by installing memoization with the command pip install --user memoization==0.4.0. Thank you very much for your attention and help!
